File: Begginers/Python/menor_e_posicao_1180.py
N = int(input())
X = input().split()
lista = [0] * N
X_int = [int(x) for x in X]

for i in range(N):  
    lista[i] = X_int[i]
    
    
menor = lista[0]
posicao = 0    
for j in range(len(lista)):
    if lista[j] < menor:
       menor = lista[j]
       posicao = j
       
print(f"Menor valor: {menor}")
print(f"Posicao: {posicao}")

File: Begginers/C++/menor_e_posicao_1180.cpp
#include <iostream>
#include <vector>

int main() {
    int N;
    scanf("%d", &N);
    
    std::vector<int> lista(N);
    for (int i = 0; i < N; i++) {
        int x;
        scanf("%d", &x);
        lista[i] = x;
    }
    
    int menor = lista[0];
    int posicao = 0;
    for (int j = 0; j < N; j++) {
        if (lista[j] < menor) {
            menor = lista[j];
            posicao = j;
        }
    }
    
    printf("Menor valor: %d\n", menor);
    printf("Posicao: %d\n", posicao);
    
    return 0;
}
